### PR TITLE
Cooper Adjustments

### PR DESCRIPTION
**Pull request by MrRowey**
Completes #4980

### CODE REVIEW

File: units/XES0102/XES0102_unit.bp [hunk 1]
```diff
@@ -128,13 +128,13 @@ UnitBlueprint {
...
-        BuildCostMass = 900,
-        BuildTime = 3600,
+        BuildCostEnergy = 6480,
+        BuildCostMass = 810,
+        BuildTime = 3240,
     },
     Footprint = {
-        SizeX = 2,
-        SizeZ = 6,
+        SizeX = 3,
```
Comment (Tagada14) [inline units/XES0102/XES0102_unit.bp:136]: Is the increase in the X dimension on purpose?

Comment (MrRowey) [reply units/XES0102/XES0102_unit.bp:136]: yes

Comment (Tagada14) [reply units/XES0102/XES0102_unit.bp:136]: Why was it necessary?

**Comment by Garanas** [comment]
@MrRowey can you add a screenshot of the old formation and the new formation, with just coopers?

**Pull request merged**